teraserver/python/modules/FlaskModule/API/service/ServiceQueryParticipants.py

```diff
@@ -11,6 +11,7 @@
 # Parser definition(s)
 get_parser = api.parser()
 get_parser.add_argument('participant_uuid', type=str, help='Participant uuid of the participant to query')
+get_parser.add_argument('participant_id', type=str, help='Participant id of the participant to query')
```

Comment:
This should be removed - when possible, service API should use UUIDs instead of IDs.